relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -445,6 +412,12 @@ fn extract_transaction_metrics_inner(
         for (breakdown, measurements) in store::get_breakdown_measurements(event, breakdowns_config)
         {
             for (measurement_name, annotated) in measurements.iter() {
+                if measurement_name == "total.time" {
+                    // The only reason we do not emit total.time as a metric is that is was not
+                    // on the allowlist in sentry before, and nobody seems to be missing it.
+                    continue;
+                }
```

Comment:
I don't understand why we should remove it. If it's not in the allowlist and a user adds it, my expectation is either to treat is as a custom measurement or to just not support it (e.g. it could be a typo).

Reply:
Sounds good, thanks for the explanation.

Out of the scope of the PR, but we should make a decision on if we want to keep it or not. If nobody misses it (or can be calculated from other metrics), we can remove it from the codebase instead of generating and discarding it.